README.md

```diff
@@ -408,6 +408,12 @@ Pyxel application file also can be converted to an executable or an HTML file wi
 - `quit()`<br>
   Quit the Pyxel application.
 
+- `tstamp()`<br>
+  Returns the number of seconds since epoch.
+
+- `ticks()`<br>
+  Returns the number of milliseconds since init.
```

Comment:
We have `frame_count`, which I think is easier to use that real time!

Reply:
`frame_count` is used for tracking time in combination with the `fps` value used at `init` time, while `ticks` would be an absolute metric of passing time. Being `fps` configurable, using `frame_count` alone is in general not accurate for tracking time.